extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -338,6 +395,20 @@ public Table readTable(
         return readTableInternal(tableIdentifier, null, instructions);
     }
 
+    /**
+     * Read the latest static snapshot of an Iceberg table from the Iceberg catalog.
+     *
+     * @param tableIdentifier The table identifier to load
+     * @param instructions The instructions for customizations while reading
+     * @return The loaded table
+     */
+    @SuppressWarnings("unused")
+    public Table readTable(
+            @NotNull final String tableIdentifier,
+            @NotNull final IcebergInstructions instructions) {
```

Comment:
I think we can make the `instructions` `@Nullable` for all of these `readTable` methods.